project-management.md

```diff
@@ -36,3 +37,23 @@ The project lifecycle is as follows:
 * Potential projects are moved to the list of **Scheduled Projects** once they have a planned start date. Having a planned start date lets potential contributors know when they need to make themselves available, and get prepared to begin their work. Subject matter experts and participants who plan to do a lot of work – such as building prototypes – benefit greatly from having a start date, as they can plan for their participation with their employers and coworkers.
 * Once a project is begun, it is moved to the list of **Current Projects**. Projects are only started when the necessary resources are available to move them quickly to completion. This means that the necessary subject matter experts have been identified, and at least two TC members are committed to review and guide the project through the specification process.
 * Once work is complete, and the working group is no longer meeting, the project document is moved to the [completed projects](projects/completed-projects/) folder.
+
+## Project Check-Ins
+
+As defined in the [charter of the GC](./governance-charter.md#regular-check-ins-with-sig-leads) the GC will check in with maintainers and project leads on a regular basis. During this check in,
+
+* any project files or roadmap entries should be reviewed to verify that they are still accurate.
+* If a SIG lead has any private concerns they wish to raise with the Governance Committee, this is an opportunity to do so.
+
+The check in process is as follows:
+
+* For each project a GC member is appointed as a liaison person. The names of that person will be listed publicly.
+* If a GC member is the maintainer/project lead themselves, they are by default the liaison person for this project.
+* The liaison person is appointed for the duration of the project lifecycle, or until they do not get reelected into the GC. Outside of that regular duration, a liaison person can resign by providing a named successor.
+* The liaison person is responsible to check in with the maintainers/project leads at least once per month. This can either happen via a (private, non recorded) meeting or through a text-based conversation initiated by the liaison person.
+* The liaison person is responsible to bring the concerns of the SIG leads forward to the GC, and they are responsible for ensuring that all action items agreed upon during that check in are logged as issues on the project repository or community repository.
```

Comment:
I tend to prefer that the SIG leads open these issues, unless it's something sensitive and they wish to remain anonymous

Reply:
I would agree in principle to that, but do we have a documented way for this to happen? Nothing in the maintainers responsibilities [here](https://github.com/svrnm/community/blob/sig-checkin-process/community-membership.md#responsibilities-and-privileges-3) mentions scaling SIG health issues to the GC (only technical issues to the GC). Should we add something there instead? Do we propose a channel for it?

Reply:
Just to clarify: I'm not saying that we only rely on SIG leads raising issues with the GC to assess health. I fully support the quarterly health update below, driven by the liaison as well as maintainers.

Reply:
re-reading, I'm good with the current language:

> responsible for ensuring that all action items agreed upon during that check in are logged as issues

"responsible for" doesn't necessarily mean logging the issue if it's something that the SIG leads can do